### PR TITLE
Fixed readme, added initial mapping data

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

**Comment by brettminnie** [comment]
Added a sql file and fixed something in the readme

**Comment by eddiejaoude** [comment]
Build failed...

```bash
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - The requested package nelmio/api-doc-bundle == 2.10.9999999.9999999-dev could not be found.
Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.
Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
Installation failed, reverting ./composer.json to its original content.
The command "composer require satooshi/php-coveralls:~0.6@stable" failed and exited with 2 during .
```

File: src/TransformCore/PHE/LocalHealthAuthorityBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -1,28 +1,25 @@
 services:
-#    transform_core_phe_local_health_authority.example:
-#        class: TransformCore\PHE\LocalHealthAuthorityBundle\Example
-#        arguments: [@service_id, "plain_value", %parameter%]
 
   transformcore.phe.lha.county:
     class: TransformCore\PHE\LocalHealthAuthorityBundle\Service\CountyService
-    arguments: [ @doctrine.orm.entity_manager ]
+    arguments: [ '@doctrine.orm.entity_manager' ]
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/LocalHealthAuthorityBundle/Resources/config/services.yml]: Are `'` needed?

Comment (brettminnie) [inline src/TransformCore/PHE/LocalHealthAuthorityBundle/Resources/config/services.yml]: Yes according to symfony 2.8, if they are missing we get an invalid yaml exception

**Pull request merged**